sharktank/sharktank/examples/paged_llm_v1.py

```diff
@@ -201,6 +201,20 @@ def pad_block_ids(self) -> torch.Tensor:
         return torch.tensor(rows, device=self.parent.model.device)
 
 
+def quantize_theta(theta):
+    if isinstance(theta, Theta) or isinstance(theta, dict):
+        if "q_input" not in (theta._tree if isinstance(theta, Theta) else theta):
```

Comment:
can probably do all this if isinstance stuff once at the beginning by assigning the result to variable, much more readable